documents/scitools_licensing.md

```diff
@@ -0,0 +1,124 @@
+# SciTools Projects
+
+## Licensing
+
+All projects under the SciTools[-incubator] GitHub organisations are to be
+developed in the open (i.e. via small pull requests which are open to
+public review) and released under an OSI (Open Source Initiative) approved
+license.
+
+
+## Copyright
+
+All SciTools projects operate a shared ownership model as specified
+in the [SciTools CLA](https://scitools.org.uk/cla/v4).
+The copyright statement is:
+
+```
+Copyright © <start year>(-<end year>) <project> contributors
+```
+
+For example:
+
+```
+Copyright © 2010-2018 Iris contributors
+```
+
+
+## Source preamble
+
+All non-trivial source files in a SciTools repository must reference the
+license of the repository. The wording of the preamble should be:
+
+```
+# Copyright © <project> contributors
```

Comment:
A unicode character here can be a problem for Python 2.7. For example, a trivial `hello_unicode.py`:

```
# Copyright © <YEAR_START>-<YEAR_END> <project> contributors
print("hello")
```
Executing with Python 2.7 we see:

```python-traceback
  File "hello_unicode.py", line 1
SyntaxError: Non-ASCII character '\xc2' in file hello_unicode.py on line 1, 
but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

It can be fixed by declaring the source encoding as utf-8. We have options:

1. Don't use this symbol
2. Use it and declare encoding on all files (yuck)
2. Use it and don't care about Python 2

For me the only practical choice is 1, especially since I'm not aware that the symbol has any legal significance in modern copyright law.

Reply:
This is a fair reflection of the situation. I agree with just dropping the © altogether (not replacing with ``(C)``).